src/semgrep_agent/semgrep.py

```diff
@@ -187,9 +187,21 @@ def get_findings(
                 for e in findings.errors:
                     for s in render_error(e):
                         click.echo(f"|    {s}", err=True)
+            inventory_findings_len = 0
+            for finding in findings.current:
+                if "__r2c-internal-cai" in finding.check_id:
+                    inventory_findings_len += 1
             click.echo(
                 f"| {unit_len(findings.current, 'current issue')} found", err=True
             )
+            click.echo(
+                f"| {unit_len(range(len(findings.current)-inventory_findings_len), 'current rule issue')}",
```

Comment:
Why is range necessary?

Reply:
unit_len requires an iterable as a parameter so unit_len(int) doesn't work but unit_len(range(int)) does